### PR TITLE
Changed preset framerates; Add "low power" preset.

### PR DESCRIPTION
**Pull request by Penguin-Guru**

### CODE REVIEW

**Comment by digisomni** [comment]
I'm on Windows 10, there's an issue where "High" gets the minimum possible resolution scale.

image

**Comment by ksuprynowicz** [comment]
I just built and tested it on Windows 10, and I'm unable to replicate the problem. Resolution scale is always 1.0 for me in presets:
obraz

**Comment by Penguin-Guru** [comment]
> I'm on Windows 10, there's an issue where "High" gets the minimum possible resolution scale.
> 
> image

I'm not sure how the changes I made could have caused this. Does it occur consistently after a clean build? No scripts that might be affecting it?

File: interface/src/PerformanceManager.cpp
Comment (daleglass) [inline interface/src/PerformanceManager.cpp:131]: Why the newline here?

File: interface/src/PerformanceManager.cpp
Comment (daleglass) [reply interface/src/PerformanceManager.cpp:116]: Interestingly git is not finding a conflict here after merging #1425, probably needs a rebase.

**Review by daleglass — CHANGES_REQUESTED** [review]

File: interface/src/PerformanceManager.cpp
Comment (daleglass) [inline interface/src/PerformanceManager.cpp:135]: Is there any reasonable way of getting here? There should either be no default for a compile-time warning, or a warning log instead.

**Review by daleglass — CHANGES_REQUESTED** [review]

**Comment by digisomni** [comment]
A fresh install with a fresh configuration yields the same results.

OS Name	Microsoft Windows 10 Pro
Version	10.0.19042 Build 19042
Adapter Description	NVIDIA GeForce RTX 3060 Ti
Driver Version	30.0.14.7196

**Comment by ksuprynowicz** [comment]
> A fresh install with a fresh configuration yields the same results.

Can you try from different Windows user account?

File: interface/src/PerformanceManager.cpp
Comment (Penguin-Guru) [reply interface/src/PerformanceManager.cpp:131]: Good question. I must have been tired or something. I'll push a fix now.

File: interface/src/PerformanceManager.cpp
Comment (Penguin-Guru) [reply interface/src/PerformanceManager.cpp:134]: I don't know what UNKNOWN was intended for. I didn't really look into it but I don't remember seeing it used anywhere. It's not even the default mode. It wasn't related to this PR though so I didn't want to make too many unnecessary changes. Might be good to try logging it for a while and see if the log ever appears? If not, then remove it?

File: interface/src/PerformanceManager.cpp
Comment (Penguin-Guru) [reply interface/src/PerformanceManager.cpp:116]: Do you want to do it or do you want me to? The name will probably need to be changed in the new case condition.

**Comment by Penguin-Guru** [comment]
> A fresh install with a fresh configuration yields the same results.
> 
> OS Name Microsoft Windows 10 Pro Version 10.0.19042 Build 19042 Adapter Description NVIDIA GeForce RTX 3060 Ti Driver Version 30.0.14.7196

So the Resolution Scale is set to minimum when you switch to HIGH (and only HIGH)? Does it change when you switch to any other preset after that?

**Comment by digisomni** [comment]
> 
> 
> > A fresh install with a fresh configuration yields the same results.
> > OS Name Microsoft Windows 10 Pro Version 10.0.19042 Build 19042 Adapter Description NVIDIA GeForce RTX 3060 Ti Driver Version 30.0.14.7196
> 
> So the Resolution Scale is set to minimum when you switch to HIGH (and only HIGH)? Does it change when you switch to any other preset after that?

Yeah, it goes back to a normal 1.0 scale for the other options.

**Comment by Penguin-Guru** [comment]
I'm compiling something to test. Will push for you to test if it works for me.

**Comment by Penguin-Guru** [comment]
Can you test with that change?

File: interface/src/PerformanceManager.cpp
Comment (daleglass) [reply interface/src/PerformanceManager.cpp:116]: Rebase it on top of the latest code, because PR #1425 will have also touched this bit, and it was merged.

**Comment by ctrlaltdavid** [comment]
Regarding the low resolution for HIGH setting, see the following line in `PerformanceManager::applyPerformancePreset()`:
```
const float RECOMMANDED_PPI[] = { 200.0f, 120.f, 160.f, 250.f};
```

**Comment by Penguin-Guru** [comment]
> Regarding the low resolution for HIGH setting, see the following line in `PerformanceManager::applyPerformancePreset()`:
> 
> ```
> const float RECOMMANDED_PPI[] = { 200.0f, 120.f, 160.f, 250.f};
> ```

Yeah, I just pushed that change to test. I'm not sure why the problem didn't occur for me or ksuprynowicz though.

File: interface/src/PerformanceManager.cpp
Comment (Penguin-Guru) [reply interface/src/PerformanceManager.cpp]: I just did. Hopefully I didn't break anything. :p

**Comment by digisomni** [comment]
Once it builds then I will give it a shot. :)

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (macOS-10.15, client)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1422/Vircadia-Interface-PR1422-9f1bdf8-_02126869.dmg

**build (macOS-10.15, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1422/Vircadia-PR1422-9f1bdf8-_02126869.dmg

**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1422/Vircadia-PR1422-9f1bdf8-.tar_02126869.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1422/Vircadia-PR1422-9f1bdf8-_02126869.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1422/Vircadia-PR1422-9f1bdf8-.tar_02126869.gz

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1422/Vircadia-PR1422-9f1bdf8-_02126869.exe

**build (self-hosted_debian-11_aarch64, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1422/Vircadia-PR1422-9f1bdf8-.tar_02126869.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1422/Vircadia-PR1422-9f1bdf8-_02126869.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1422/Vircadia-PR1422-9f1bdf8-.tar_02126869.gz

**Comment by digisomni** [comment]
The issue appears to be solved now. Can anyone else download a copy from GH and check to confirm as well?

**Comment by ksuprynowicz** [comment]
> 
> 
> The issue appears to be solved now. Can anyone else download a copy from GH and check to confirm as well?

I just tested it on Windows 10, and it works perfectly, both in VR and in Desktop mode

File: interface/src/PerformanceManager.cpp
Comment (daleglass) [reply interface/src/PerformanceManager.cpp:134]: Yeah, I think a log would be good here. I don't think anything should be reaching this point legitimately.

**Review by daleglass — APPROVED** [review]

**Comment by Penguin-Guru** [comment]
I could add the logging now but I wouldn't be able to test it for several days and it's kind of outside the scope of this P.R. I think it would be better if someone makes the change outside of this P.R. If you do want it here, feel free to make the change yourself. Edits by maintainers is enabled.

**Pull request merged**
